brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
It looks like all the things you said in your comment still apply to this method, so I guess the question is more if it is resolved in your mind than if it is resolved in the code.